includes/snmp.inc.php

```diff
@@ -465,6 +465,11 @@ function snmpwalk_cache_oid($device, $oid, $array, $mib = null, $mibdir = null,
 {
     $data = snmp_walk($device, $oid, $snmpflags, $mib, $mibdir);
     foreach (explode("\n", $data) as $entry) {
+        if (! Str::contains($entry, ' =') && ! empty($entry)) {
+            $array[$index][$oid] .= "\n$entry";
```

Comment:
`$index` is undefined here

Reply:
Yes indeed, but the block should never be entered in the first loop.